.github/workflows/ci.yml

```diff
@@ -117,6 +118,15 @@ jobs:
     - name: Check bless provider
       if: github.event == 'push' || steps.filter.outputs.module == 'true'
       run: ls -al $HOME/.terraform.d/plugins/terraform-provider-bless*
+
```

Comment:
hmm maybe we should just run `make setup` rather than duplicate this.

Reply:
adding a follow-up pr for this